Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lighting/LightingData.azsli

```diff
@@ -13,33 +13,41 @@
 #include <Atom/Features/LightCulling/LightCullingTileIterator.azsli>
 #include <Atom/Features/PBR/LightingUtils.azsli>
 
+#ifndef MAX_SHADING_VIEWS
+#define MAX_SHADING_VIEWS 1
+#endif
+
+#ifndef GET_SHADING_VIEW_COUNT
```

Comment:
Nit. I'd call this `SHADING_VIEW_COUNT`. When I see GET_xxx without parentheses feels like a programmer error for not having `()`.